TKX-1969 Upgrade to .NET 8

emanuelgaspar opened the pull request:
Also refresh Api client with NSwag 14.0.2

emanuelgaspar committed:
upgrade to .NET 8

emanuelgaspar committed:
refresh api client with NSwag 14.0.2

joaofbvieira reviewed (commented):
Please check the `AuthorisedClient` in API Client project that should use `BaseUrl.AbsoluteUri` to ensure the uri is properly parsed and ends with "/".
Reference: https://github.com/trakx/coingecko-api-client/blob/d562899ec3f3edae452970e32aa3ee5e009acab2/src/Trakx.CoinGecko.ApiClient/Clients/AuthorisedClient.cs#L9

emanuelgaspar committed:
use AbsoluteUri to ensure slash at end ( thanks @joaofbvieira )

emanuelgaspar commented:
> Please check the `AuthorisedClient` in API Client project that should use `BaseUrl.AbsoluteUri` to ensure the uri is properly parsed and ends with "/". Reference:
> 
> https://github.com/trakx/coingecko-api-client/blob/d562899ec3f3edae452970e32aa3ee5e009acab2/src/Trakx.CoinGecko.ApiClient/Clients/AuthorisedClient.cs#L9

Thanks, changed and also added a unit test.
